src/components/views/elements/ImageView.js

```diff
@@ -168,15 +184,26 @@ module.exports = React.createClass({
             </div>);
         }
 
+        const rotationDegrees = this.state.rotationDegrees;
+        const effectiveStyle = {transform: `rotate(${rotationDegrees}deg)`, ...style};
+
         return (
             <div className="mx_ImageView">
                 <div className="mx_ImageView_lhs">
                 </div>
                 <div className="mx_ImageView_content">
-                    <img src={this.props.src} title={this.props.name} style={style} />
+                    <img src={this.props.src} title={this.props.name} style={effectiveStyle} className="mainImage" />
                     <div className="mx_ImageView_labelWrapper">
                         <div className="mx_ImageView_label">
-                            <AccessibleButton className="mx_ImageView_cancel" onClick={ this.props.onFinished }><img src={require("../../../../res/img/cancel-white.svg")} width="18" height="18" alt={ _t('Close') } /></AccessibleButton>
+                            <AccessibleButton className="mx_ImageView_rotateCounterClockwise" onClick={ this.rotateCounterClockwise }>
+                                <img src={require("../../../../res/img/rotate-ccw.svg")} alt={ 'Rotate counter-clockwise' } width="18" height="18" />
```

Comment:
The alt needs to be translatable: `_t('Rotate counter-clockwise')` and `yarn i18n`

Applies to the other image as well.